sig-auth/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to SIG Auth
+
+Welcome to contributing to SIG Auth.
+
+If you haven't seen them already, the Kubernetes project has:
+
+- A [Contributor Guide][contrib-guide] - some
+  [kubernetes/kubernetes]-specific content, but lots of info for the
+  entire project
+- A [Contributor Cheat Sheet][contrib-cheatsheet] - lots of resources
+  and handy links
+
+SIG Auth has multiple areas you can contribute to. Those contributions
+can be in the form of code, documentation, support being involved in
+mailing list discussions, attending meetings, and more. This guide
+describes different major functional areas SIG Apps is involved in,
+provides an overview of the areas, and gives pointers on getting more
+involved in each area. Consider this a launching point or the start of
+a [choose your own
+adventure](https://en.wikipedia.org/wiki/Choose_Your_Own_Adventure)
+for SIG Auth.
+
+## Workflow
+
+Just like rest of the Kubernetes project, we also use the same PR and
+review based workflow. Which means [use of the CNCF CLA][cla], [code
+review by reviewers and approvers listed in OWNERS files][owners], and
+tests that automatically exercise code or enforce conventions.
+
+Some other points to keep in mind while contributing to this repo:
+
+- For large code changes, please write a design doc or [KEP] and get
+  signoff from SIG Auth before trying to land code. If you're not sure
+  what a KEP should look like,
+  [kuberenetes/enhancements/keps/sig-auth] has some examples. If
+  you're not sure what "large" means, [come ask us](#contact)
+- We find it polite to apply the `do-not-merge/hold` label via the `/hold`
+  command when in doubt about whether/when a PR should merge. If we don't
+  explain why we're adding a `/hold`, this is usually because the PR will
+  cause changes to be deployed on merge, and we want the person responsible
+  for deploying and monitoring the changes to decide when to `/hold cancel`.
+  In some cases this is the PR author, in other cases this may be whomever
+  is on-call for prow.k8.io.  If you are unsure which, please ask.
```

Comment:
this doesn't really apply to sig-auth PRs... this sounds like it was copied from a test-infra contributing doc